src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java15/InputAstRegressionSealedAndPermits.java

```diff
@@ -0,0 +1,34 @@
+//non-compiled with javac: Compilable with Java15
+package com.puppycrawl.tools.checkstyle.grammar.java15;
+
+public sealed class InputAstRegressionSealedAndPermits permits Circle, Square, Rectangle {
```

Comment:
We have a check to look to extend this token to existing checks?

Does ModifierOrder handle this correctly? or any of the other modifier checks?

Reply:
I've added reports above, and opened https://github.com/checkstyle/checkstyle/issues/9854 in response to the false positives by ModiferOrder. No exceptions were found during regression on this branch.

Reply:
Thanks. I just wanted to note that the reports were regression, they will not show false negatives. Only false positives or exceptions.

My main concern was the modifier checks. Otherwise, I am not sure what other checks would be impacted.

I am marking this as resolved.